examples/AspNet/Global.asax.cs

```diff
@@ -55,5 +76,44 @@ protected void Application_End()
         {
             this.tracerProvider?.Dispose();
         }
+
+        private class DebugStreamWriter : StreamWriter
```

Comment:
The .NET Framework example is a web application so I added this to redirect console output to the VS debug window.

Reply:
Might be a good addition to the ConsoleExporter.

Reply:
@reyang You thinking something like adding an option to have ConsoleExporter switch to Debug.Write instead of Console.Write?

Reply:
Yes, I see a general need for processes that don't have a "console", and I think having yet another "Debug stream exporter" might be an overkill.
In C++ we have "ostream exporter" which allows whatever output stream, and the default is STDOUT. And I think it is a good approach.

Reply:
#1449